auction_server/server.js

```diff
@@ -0,0 +1,39 @@
+import express from 'express';
+
+const pool = require("../db");
+
+const app = express();
+
+app.use(express.json()); 
+
+
+//ROUTES/
+
+app.get("/api/items", async(req, res) => {
+    try {
+        const allItems = await pool.query("SELECT * FROM items");
+        res.json(allItems.rows);
+    } catch (err) {
+        console.log(err.message);
+    }
+})
+
+app.get("/api/users", async(req, res) => {
+    try {
+        const allItems = await pool.query("SELECT * FROM items");
+        const allUsers = await pool.query("SELECT * FROM users");
+        const allBids = await pool.query("SELECT * FROM bids");
+        
+        res.json({
+            items: allItems.rows,
```

Comment:
It is extremely bad practice to return **all** items from API (unless it is always small amount of items, but this is not the case).
Other than that, api `/api/users` should **not** return items and bids. You should have separate api to return those.
Other than that, filtering and sorting are not implemented whatsoever.